### PR TITLE
wknapik generic testing 01

### PR DESCRIPTION
**Pull request by wknapik**
PLEASE DO NOT REVIEW, THIS IS JUST A TEST PR

Resolves nothing

## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: components/brave_rewards/browser/test/rewards_publisher_browsertest.cc [hunk 1]
```diff
@@ -43,7 +43,7 @@ class RewardsPublisherBrowserTest : public InProcessBrowserTest {
     https_server_->SetSSLConfig(net::EmbeddedTestServer::CERT_OK);
     https_server_->RegisterRequestHandler(
         base::BindRepeating(&rewards_browsertest_util::HandleRequest));
-    ASSERT_TRUE(https_server_->Start());
+    ASSERT_FALSE(https_server_->Start());
```
Comment (emerick) [inline components/brave_rewards/browser/test/rewards_publisher_browsertest.cc:46]: Sorry, couldn't tell... Are these changes intentional?

Comment (wknapik) [reply components/brave_rewards/browser/test/rewards_publisher_browsertest.cc:46]: Yeah, I'm intentionally breaking tests to test changes in pipelines. This PR is not meant to be reviewed, or merged. I'm not sure how to indicate that, so people don't waste time looking at it. Figured the PR name would suffice...

I see github is automatically adding reviewers and I can't remove them ;/

**Comment by wknapik** [comment]
Seems CHROMIUM_MAJOR is set to `null` in [brave-browser-common.groovy:694](https://github.com/brave/devops/blob/chromium-brave-core/jenkins/jobs/browser/brave-browser-common.groovy#L694), judging from [this failure](https://ci.brave.com/job/pr-brave-browser-wknapik-generic-testing01-windows/13/execution/node/273/log/):
image

**Review by fmarier — CHANGES_REQUESTED** [review]
Denied! :stuck_out_tongue_closed_eyes:
